jgiven-gradle-plugin/src/main/java/com/tngtech/jgiven/gradle/internal/JGivenHtmlReportImpl.java

```diff
@@ -4,7 +4,7 @@
 import com.tngtech.jgiven.report.ReportGenerator;
 import org.gradle.api.Task;
 
-public class JGivenHtmlReportImpl extends AbstractJGivenReportImpl implements JGivenReport {
+public abstract class JGivenHtmlReportImpl extends AbstractJGivenReportImpl implements JGivenReport {
```

Comment:
I cannot do a concrete implementation here, because gradle wants to call the method `getObjectLocation()` before my field `objectLocation` has even the chance to be initialized. 
however, as @wolfs  pointed out to me (thanks again) Gradle can instantiate these abstract classes without any problem